Sources/ssg/SwiftStringGarbler+Extensions.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  SwiftStringGarbler+Extensions.swift
+//  ssg
+//
+//  Created by Norman Barnard on 5/5/21.
+//
+
+import Foundation
+import TSCBasic
+import CryptoKit
+
+public extension AbsolutePath {
+    func existsAsFile(in fileSystem: FileSystem = localFileSystem) -> Bool {
+        return fileSystem.isFile(self)
+    }
+}
+
+public extension String {
+    @available(OSX 10.15, *)
+    func sha256Checksum() -> String {
+        guard let d = data(using: .utf8) else { fatalError("Can't get data representation of string \(self)") }
```

Comment:
Nitpick: I would rather see `d` spelled out to something more meaningful such as `data`.